nuget.config

```diff
@@ -1,7 +1,7 @@
 <configuration>
   <packageSources>
-    <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="PullRequests" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-PullRequests/nuget/v3/index.json" protocolVersion="3" />
```

Comment:
I thought this was reverted? This is causing the packaging step error, we shouldn't have this here.

Reply:
Good catch, this shouldn't be here anymore.